docs/exp_hardcode.md

```diff
@@ -5,6 +5,16 @@ sidebar_position: 4
 
 ![Whalepass.gg](https://i.imgur.com/zwUqWaS.png)
 
+This table outlines the token types and body fields required for API requests, their locations, formats, and where to find or provide the necessary values.
+
+| Token Type   | Location         | Format                               | Where To Find                              |
+|:------------:|:----------------:|--------------------------------------:|-------------------------------------------:|
+| Header Token | X-API-KEY        | X-API-KEY: YOURTOKEN                 | https://dashboard.whalepass.gg/api-key     |
+| Header Token | X-Battlepass-Id  | X-Battlepass-Id: YOURTOKEN           | https://dashboard.whalepass.gg/campaigns   |
+| Path Field   | playerId         | { "playerId": "string" }             | You can find in response                   |
+| Body Field   | gameId           | { "gameId": "string" }               | https://dashboard.whalepass.gg/game-actions|
```

Comment:
Here is the url is wrong for gameId, we can put the url of how to find your game id from the tutorial